release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1108,7 +1108,11 @@ They correspond to older, similarly named SCCs, but with the following enhanceme
 * `seccompProfile` is defaulted to `runtime/default` if unset. In previous releases, this field had to be empty.
 * `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. Previously, a `true` value was allowed.
 
-New clusters allow use of the `restricted-v2` SCC for any authenticated user in place of the `restricted` SCC; the `restricted` SCC is no longer available to users of new clusters, unless the access is explicitly granted. Clusters that are upgraded to {product-title} 4.11 from previous versions allow both `restricted-v2` and `restricted` SCCs.
+The default SCC of the cluster depends on whether the cluster is a new {product-title} 4.11 installation or an upgraded cluster.
```

Comment:
Similarly as in the other PR - "default" is not very explanatory here, and I would perhaps even say it is wrong as per the below description.

See my comment here: https://github.com/openshift/openshift-docs/pull/49188#discussion_r951391625